target/braintree-1.3.10.js

```diff
@@ -2366,13 +2366,11 @@ sjcl.random = {
       }
     }
 
-    const forbiddenKeys = ['__proto__', 'constructor', 'prototype'];
     for (i=0; i<jsTemp.length; i++) {
       j = jsTemp[i];
-      if (forbiddenKeys.includes(j)) {
-        continue;
+      if (!['__proto__', 'constructor', 'prototype'].includes(j)) {
```

Comment:
from what I understand in this project, these files are from a third-party library (idk why we don't use node_modules). Is it a good idea to change them?

Reply:
I think we are maintaining a fork of Braintree so we either have to patch our fork or update it from upstream... I don't think we are likely to un-fork though. I also don't know why we have the fork. It would be nice if that was documented.

Ah I see that this is a completely deprecated package... so I guess that has something to do with it? https://github.com/github/braintree-encryption/commit/a1c4125cfd9ad6be7b470c66b1faf80b0f8f7d3e